src/app/pages/journal/journal.page.ts

```diff
@@ -81,17 +81,17 @@ export class JournalPage extends BasePageComponent implements OnInit {
   subscription: Subscription;
   employeeId: string;
   start = '2018-12-10T08:10:00+00:00';
-  merged$: Observable<void | number>;
+  merged$: Observable<void | number | Promise<void>>;
```

Comment:
should not be mixing Observable/Promises here, at least no other the part has needed to do this

Reply:
will require a re-write of code.